challenge-react/src/hooks/useCharity/useCharity.test.js

```diff
@@ -0,0 +1,40 @@
+import  * as reactRedux from 'react-redux'
+
+import useCharity from './useCharity'
+import charityAction from '~modules/charity/charityAction'
+import createStore from '~root/createStore'
+
+const state = createStore().getState()
+
+beforeEach(() => {
+    reactRedux.useSelector.mockImplementation(cb => cb(state))
+})
+
+describe('useCharity', () => {
```

Comment:
Somehow I run this test file locally, and it's failed. So I think you configured babel wrong.